src/test/ui/privacy/issue-79593.stderr

```diff
@@ -10,11 +10,17 @@ error[E0063]: missing field `y` in initializer of `Enum`
 LL |         Enum::Variant { x: () };
    |         ^^^^^^^^^^^^^ missing `y`
 
-error: cannot construct `Pub` with struct literal syntax due to inaccessible fields
+error: cannot construct `Pub` with struct literal syntax due to private fields
   --> $DIR/issue-79593.rs:18:5
    |
 LL |     foo::Pub {};
    |     ^^^^^^^^
+   |
+note: missing field `private` is private
+  --> $DIR/issue-79593.rs:2:22
+   |
+LL |     pub struct Pub { private: () }
```

Comment:
Hm, maybe we shouldn't actually point out private fields' definition. It seems like excessive info and showing a private field is kinda against its own purpose...